vm/chef/cookbooks/rabbitmq/files/rabbitmq

```diff
@@ -89,10 +89,12 @@ if [[ "${is_stats_node:-0}" -ne 0 ]]; then
   rabbitmqctl set_permissions "${rabbitmq_username}" ".*" ".*" ".*"
   rabbitmqctl set_user_tags "${rabbitmq_username}" administrator
 
-  readonly rabbitmq_admin_cmd="$(find /var/lib/rabbitmq/* -name rabbitmqadmin)"
-  chmod u+x "${rabbitmq_admin_cmd}"
+  # Download Management CLI
+  # https://www.rabbitmq.com/management-cli.html
+  curl -o /usr/sbin/rabbitmqadmin http://localhost:15672/cli/rabbitmqadmin \
```

Comment:
Is this guaranteed to always return the same version of `rabbitmqadmin`. If not, I think we should try to ensure this is as hermetic as possible and download the github version instead

Reply:
Yes. In fact, that's an administration CLI which comes embedded with rabbitmq. It should be always compatible with current rabbitmq version. 

https://www.rabbitmq.com/management-cli.html